### PR TITLE
#15: hid EXA_RLS_USERS table from metadata

### LINKED ISSUE #15: Hide EXA_RLS_USERS in VS metadata
## Situation

Regular users should not be able to see other users' role assignements.

## Acceptance Criteria

1. `EXA_RLS_USERS` inaccessible for users through RLS VS

### PR DESCRIPTION
**Pull request by AnastasiiaSergienko**
Closes #15

### CODE REVIEW

File: src/main/java/com/exasol/adapter/dialects/rls/RowLevelSecurityQueryRewriter.java [hunk 1]
```diff
@@ -56,7 +55,7 @@ private String rewriteStatement(final SqlStatement statement, final ExaMetadata
         final SqlStatementSelect select = (SqlStatementSelect) statement;
         final String schemaName = properties.getSchemaName();
         final UserInformation userInformation = new UserInformation(exaMetadata.getCurrentUser(), schemaName,
-                "EXA_RLS_USERS");
+                EXA_RLS_USERS_TABLE_NAME);
```
Comment (redcatbear) [inline src/main/java/com/exasol/adapter/dialects/rls/RowLevelSecurityQueryRewriter.java:58]: :+1:

File: src/test/java/com/exasol/adapter/dialects/rls/RowLevelSecurityDialectIT.java [hunk 2]
```diff
@@ -6,10 +6,15 @@
 import static org.junit.jupiter.api.Assertions.assertThrows;
 
 import java.nio.file.Path;
-import java.sql.*;
+import java.sql.Connection;
```
Comment (redcatbear) [inline src/test/java/com/exasol/adapter/dialects/rls/RowLevelSecurityDialectIT.java]: Please correct this manually and I will check why the formatter settings are off again.

File: src/test/java/com/exasol/adapter/dialects/rls/RowLevelSecurityTableMetadataReaderTest.java [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+    @BeforeEach
+    void setUp() {
+        this.metadataReader = new RowLevelSecurityTableMetadataReader(null, null, AdapterProperties.emptyProperties(),
+                null);
+    }
+
+    @Test
+    void testIsTableIncludedByMappingTrue() {
+        assertTrue(this.metadataReader.isTableIncludedByMapping("MY_TABLE"));
```
Comment (redcatbear) [inline src/test/java/com/exasol/adapter/dialects/rls/RowLevelSecurityTableMetadataReaderTest.java]: Use hamcrest matcher please.

File: src/test/java/com/exasol/adapter/dialects/rls/RowLevelSecurityTableMetadataReaderTest.java [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+    }
+
+    @Test
+    void testIsTableIncludedByMappingFalse() {
+        assertFalse(this.metadataReader.isTableIncludedByMapping("EXA_RLS_USERS"));
```
Comment (redcatbear) [inline src/test/java/com/exasol/adapter/dialects/rls/RowLevelSecurityTableMetadataReaderTest.java]: Same here.

**Review by redcatbear — CHANGES_REQUESTED** [review]

File: src/test/java/com/exasol/adapter/dialects/rls/RowLevelSecurityDialectIT.java (hunk 2 above)
Comment (AnastasiiaSergienko) [reply src/test/java/com/exasol/adapter/dialects/rls/RowLevelSecurityDialectIT.java]: done

File: src/test/java/com/exasol/adapter/dialects/rls/RowLevelSecurityTableMetadataReaderTest.java (hunk 3 above)
Comment (AnastasiiaSergienko) [reply src/test/java/com/exasol/adapter/dialects/rls/RowLevelSecurityTableMetadataReaderTest.java]: done

File: src/test/java/com/exasol/adapter/dialects/rls/RowLevelSecurityTableMetadataReaderTest.java (hunk 4 above)
Comment (AnastasiiaSergienko) [reply src/test/java/com/exasol/adapter/dialects/rls/RowLevelSecurityTableMetadataReaderTest.java]: done

**Review by redcatbear — APPROVED** [review]

**Pull request merged**